packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -130,6 +130,24 @@ class PageStorageBucket {
 }
 
 /// A widget that establishes a page storage bucket for this widget subtree.
+///
+/// {@tool sample}
+///
+/// This sample creates a page storage for MyHomePage
```

Comment:
Can you make this example more informative? Currently, this just shows how to instantiate the class. Maybe also incorporate accessing the PageStorage from a descendant widget in the example?

Reply:
I have added read and write data sampel code